package.json

```diff
@@ -119,27 +119,28 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-vue": "^9.2.0",
     "husky": "^6.0.0",
-    "jest": "27.5.1",
-    "jest-preset-angular": "11.1.2",
+    "jest": "28.1.3",
+    "jest-environment-jsdom": "28.1.1",
+    "jest-preset-angular": "12.2.2",
     "lint-staged": "^10.5.4",
-    "ng-packagr": "14.1.0",
-    "ngx-deploy-npm": "=4.1.4",
-    "nx": "14.5.1",
-    "nx-vue3-vite": "^0.16.0",
+    "ng-packagr": "14.2.1",
+    "ngx-deploy-npm": "^4.3.0",
+    "nx": "15.0.1",
+    "nx-vue3-vite": "^0.19.0",
     "postcss": "^8.4.14",
     "postcss-import": "14.1.0",
     "postcss-preset-env": "7.5.0",
     "postcss-url": "10.1.3",
     "prettier": "2.7.1",
     "stylelint": "^14.9.1",
     "stylelint-config-standard": "^26.0.0",
-    "ts-jest": "27.1.4",
-    "ts-node": "10.8.1",
+    "ts-jest": "28.0.8",
+    "ts-node": "10.9.1",
     "tsconfig": "^7.0.0",
     "tslib": "^2.4.0",
-    "typescript": "4.7.3",
+    "typescript": "4.8.4",
     "unplugin-vue-components": "^0.19.6",
-    "vite": "^2.9.12",
```

Comment:
Is this change a concern @ktrz ?

I just updated everything I could, I didn't think much about the major version differences.

Reply:
We would need to test if it doesn't break anything for the vue-route-guard library

Reply:
Tests pass, but would you like to do something manual before merging?

Reply:
@ktrz added a build task in actions to catch things (like this PR) that break the build command